scania-digital-design-system/sdds
# fix(datetime): cancelable events and sdds-icon

**Describe pull-request**  
Set cancelable: false for non cancelable events. Also made sure we are using our own icons components for icons. Also added missing color for disabled icon in light mode.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1093

**How to test**  
1. Go to storybook link below
2. Check in Components -> Datetime -> All stories
3. Check that the icon is correct in all states.

**Suggested test steps**
